### ISSUE TITLE
ip总数到了3000上不去了

### ISSUE THREAD

**Issue by zybxqm** [XX-net/XX-Net#4379]
就是ip总数一直3000上不去

**Comment by zybxqm** [comment]
_
![qq 20161016160939](https://cloud.githubusercontent.com/assets/22865908/19416136/f8123fa2-93ba-11e6-8e76-a37bb481d7cf.png)
_

**Comment by zybxqm** [comment]
总数一直3000有效ip到了2998然后就一直徘徊

**Comment by zou888666** [comment]
软件设定，IP总数为３０００，所以不能超过３０００，但超过时，会检查已有IP，将延迟高的无效的取代，所以你这是正常的——我这么说，大家都懂吧？！不用谢我，请叫我雷锋。

**Comment by zybxqm** [comment]
可不可以改！我看别人都有7000个ip的

**Comment by yaoguaishou** [comment]
ip数量高了除了好看没别的用
你可以新建/data/gae_proxy/manual.ini
里面写上
`[google_ip]
max_good_ip_num = 999999999999999999999999999999999999999999999999999999`

**Comment by zou888666** [comment]
我会告诉你，我的IP总数只有５００吗？为什么？——因为IP的选择，是按照延迟决定的，延迟低的会将高的取消，所以IP总数越少，IP质量越高，速度越快，IP总数越多，你好羡慕，但你了解别人IP过万速度却慢如蜗牛还时不进断线的苦恼吗？

**Comment by nly4lov** [comment]
@zou888666 咋限制为500的？

**Comment by zou888666** [comment]
上面不是有吗，将那么多的９改成５００即好。

**Comment by nly4lov** [comment]
我说怎么翻半天也没找到这个设置所在的文档，要自己手动添加啊。。。

**Comment by nly4lov** [comment]
貌似是编写了这样的一个配置文件后，也需要一些时间，ip数量才会降到500

**Comment by jzp820927** [comment]
慢慢删除高延时 IP，肯定是需要时间的

**Comment by nly4lov** [comment]
@jzp820927 不对，刚才我又看了看，数字又回到3000去了啊![qq 20161009093240](https://cloud.githubusercontent.com/assets/11325411/19427839/07e0ac02-9479-11e6-9a00-8c4c56d8da69.gif)

**Comment by jzp820927** [comment]
你是不是改了没保存？或者没重启 xxnet

**Comment by nly4lov** [comment]
我是新建的文本文档，改名叫manual.ini，里边内容是[google_ip] max_good_ip_num = 500，桌面右下角右击XXNET图标退出，又启动之后，仍然是3000个不动了啊，哪不对了呢

**Comment by jzp820927** [comment]
是不是隐藏了扩展名，实际文件名是 manual.ini.txt ?

**Comment by nly4lov** [comment]
不是![qq 20160529140801](https://cloud.githubusercontent.com/assets/11325411/19459953/b8a2c0bc-9509-11e6-9531-ffc8b2c08d20.png)

**Comment by wenliyang** [comment]
这样试下

退出 XX-Net

在 XX-Net\data\gae_proxy\ 下新建 manual.ini，内容

[google_ip]
max_good_ip_num = 500

再运行 XX-Net

**Comment by nly4lov** [comment]
@wenliyang 已照做，正在观察

**Comment by nly4lov** [comment]
这个办法起作用啦！IP总数嗖地一下，就变成500了。哈哈，我隐约明白了这原因，只是说不清楚。
不过话说我也没觉得这500比3000的时候快啊，延迟倒变成 395了，原来是300左右![e103](https://cloud.githubusercontent.com/assets/11325411/19460426/3372588a-950e-11e6-9ca7-4bdc3e5bf9a8.gif)

**Comment by jzp820927** [comment]
得慢慢筛选啊

**Comment by nly4lov** [comment]
嗯，那刚刚一下就从3000筛选到500了倒是快捷得很

**Comment by jzp820927** [comment]
具体的机制不太清楚。每个人的网络环境不一样，效果也不一样。改小IP数量至少可以减少扫描压力。不一定就可以提升速度。还是和本地网络有关。

**Comment by nly4lov** [comment]
现在IP延迟只有180多了，确实感觉出快了啊，哈哈。谢谢大家的热心帮助

**Comment by chemfly** [comment]
真是快！我的设置后再扫一下已有IP，剩下250多个，延迟80多！原来300多的延迟！

**Comment by CHN008** [comment]
我按照你们说的改了,检查完所有IP后发现延迟还是一样,可能比以前还高一点儿.又改回原来的算了. ![image](https://cloud.githubusercontent.com/assets/16646697/19769355/abff4ec6-9c8d-11e6-861e-c9df3235737d.png)

**Comment by jzp820927** [comment]
主要还是看 IP 吧，重新扫描只是剔除一些不稳定的IP。

**Comment by CHN008** [comment]
哦,那还是算了,不折腾了.

发现这个网站不用代理速度很慢,开启全局代理速度才算正常.

不知最近它们又在搞什么...

**Comment by jzp820927** [comment]
我这儿是电信网络，直连访问 github 经常都是白天好，晚上卡成翔。
